packages/mermaid/src/diagrams/class/parser/classDiagram.jison

```diff
@@ -127,6 +128,10 @@ line was introduced with 'click'.
 <*>\-                           return 'MINUS';
 <*>"."                          return 'DOT';
 <*>\+                           return 'PLUS';
+":"                             return 'COLON';
+","                             return 'COMMA';
+\#                              return 'BRKT';
+"#"                             return 'BRKT';
```

Comment:
Is `BRKT` a common abbreviation of `#`? I was only aware of Hash and Pound.

Reply:
This might be a dumb question, since I'm not super experienced with JISON, but aren't both of these lines identical?

`\#` should be equivalent to `"#"`

---

Also, fun fact of the day (since I just learnt this too), apparently the `#` sign comes from `℔` which comes from lb, which is what Americans use as a unit of mass!

And you can also call it octothorpe if you want to be really fancy :)

Reply:
I never tested without the second use of BRKT. I was just making sure it matched flow.jison.  Which is also why it is called BRKT. Otherwise, I would have called it HASH

Reply:
Ah, that's fair :) In that case, we can leave it how it is then if `flow.jison` is using it, maybe in the future we can remove them.

Other than that (and the failing CI) the code for this PR looks good! Please un-draft this PR once CI is fixed and this PR is ready for review so that I can do some manual testing and see if I can break anything.